course-fund-request-cryptominded.md

```diff
@@ -0,0 +1,114 @@
+# Request for Nest membership and funding
```

Comment:
Actually can you change the directory of the file and also split into team.md and roadmap.md? Reference proposal: https://github.com/aragon/nest/pull/2/files